bubblewrap.c

```diff
@@ -2014,8 +2027,8 @@ main (int    argc,
         opt_unshare_user = TRUE;
     }
 
-  if (argc == 0)
-    usage (EXIT_FAILURE, stderr);
+  if (!opt_exec_filename)
+      die ("--exec-filename not specified");
```

Comment:
```
if (argc == 0)
  usage (EXIT_FAILURE, stderr);

if (!opt_exec_filename)
  opt_exec_filename = argv[0];
```

Otherwise, the ways people currently run `bwrap` will fail, and in particular the unit tests and Flatpak will cease to work. Command-lines are an "API" that need to be kept compatible.

The reason I say you need to keep the `if (argc == 0)` check is that executing something with an entirely empty `argv` would be bad. `argv` is meant to contain at least one element, namely `COMMAND`.